marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/jobInfo/service.json

```diff
@@ -0,0 +1,4 @@
+{
+  "endpointDirectory": "/data-hub/5/data-services/jobInfo/",
+  "$javaClass": "com.marklogic.hub.explorer.job.JobInfo"
```

Comment:
The current path resolves in the explorer project.
Please suggest a path if DHF also wants to generate the endpoint Java class.

Reply:
Similar to how we don't want "explorer" in the path of the ML module, I don't think we want it in the Java class either, as we don't expect our data services to be specific to the Explorer app. 

While I normally think that Java packages should be organized around features, I think the set of interfaces generated for data services is an exception to that. I think the organization would be better if all such interfaces were in a single package so that it's easy to quickly inspect all such interfaces to MarkLogic. 

For that reason, I think "com.marklogic.hub.dataservices" would be a good package to use. I don't think a child package under that is necessary for a data service unless we expect there to be multiple classes associated with a particular data service.

Reply:
Included in https://github.com/marklogic/marklogic-data-hub/pull/3011/commits/1720b70725138f9db551e7fb8897364f408f8c76